app/frontend/src/features/communities/CommunityPage/EventCard.tsx

```diff
@@ -2,18 +2,48 @@ import {
   Card,
   CardActionArea,
   CardContent,
+  CardMedia,
   makeStyles,
   Typography,
 } from "@material-ui/core";
 import React from "react";
 import LinesEllipsis from "react-lines-ellipsis";
 import { Link } from "react-router-dom";
 
-import { CalendarIcon, ClockIcon } from "../../../components/Icons";
+import {
+  CalendarIcon,
+  ClockIcon,
+  LocationIcon,
+} from "../../../components/Icons";
 import { routeToEvent } from "../../../routes";
 import { timestamp2Date } from "../../../utils/date";
+import eventImagePlaceholder from "./eventImagePlaceholder.svg";
 
 const useStyles = makeStyles((theme) => ({
+  link: { textDecoration: "none", color: "inherit" },
+  image: {
+    backgroundColor: theme.palette.grey[200],
+    height: 80,
+    objectFit: "contain",
+    [theme.breakpoints.up("sm")]: {
+      height: 100,
```

Comment:
Hmm might be better to use % for these height so it scales with the card's height?

Reply:
That would be equivalent to "auto" because the parent doesn't have specified height

Reply:
Uhh the pain of height not inheriting properly... I guess this will become a decorative image eventually so it's probably not as big of a deal